web/services/store/index.js

```diff
@@ -0,0 +1,50 @@
+import { createStore, combineReducers, compose } from 'redux';
+import { reactReduxFirebase, firebaseReducer } from 'react-redux-firebase';
+import firebase from 'firebase';
+
+const prodConfig = {
+  // TODO: setup production firestore project
+};
+
+const devConfig = {
+  apiKey: 'AIzaSyCBkQHeikIsiYZ2yOHiqH_mGJKDWMDU500',
```

Comment:
still iffy on leaving this here but oh well

Reply:
🤷‍♀️